src/plugins/CMakeLists.txt

```diff
@@ -12,7 +12,6 @@ function(add_plugin)
 
     add_library(${PARSED_ARGS_NAME} SHARED ${PARSED_ARGS_SRCS})
 
-    # For provider.hpp
     target_include_directories(${PARSED_ARGS_NAME} PRIVATE ${CMAKE_BINARY_DIR})
```

Comment:
Not pretty much into CMake, but with the comment I'd expect that the line is not necessary anymore